components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/java/org/wso2/carbon/mediator/enrich/ui/EnrichMediator.java

```diff
@@ -34,7 +34,6 @@
 import javax.xml.stream.XMLStreamReader;
 import java.util.Iterator;
 
-
 public class EnrichMediator extends AbstractMediator {
```

Comment:
Add class comments (javadoc). Also need to update the license header year range.

Reply:
Done